### PR TITLE
feat: add issu tag into the token with the value true

### PR DESCRIPTION
**Pull request by msimionati**
the issu tag is added with the value true in the generation of the token, the corresponding test is added to

### CODE REVIEW

File: DopplerJobTest/TokenGeneratorTests.cs [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+            // Act
+            var jwtToken = sut.CreateJwtToken();
+
+            var jsonToken = handler.ReadToken(jwtToken);
+            var tokens = handler.ReadToken(jwtToken) as JwtSecurityToken;
+            var isSuClaimValue = tokens.Claims.First(claim => claim.Type == "isSU").Value;
+
+            // Assert
+            Assert.True(isSuClaimValue == "true");
```
Comment (andresmoschini) [inline DopplerJobTest/TokenGeneratorTests.cs:38]:
```suggestion
            // Assert
            var jsonToken = handler.ReadToken(jwtToken);
            var tokens = handler.ReadToken(jwtToken) as JwtSecurityToken;
            var isSuClaimValue = tokens.Claims.First(claim => claim.Type == "isSU").Value;
            Assert.True(isSuClaimValue == "true");
```

**Comment by andresmoschini** [comment]
👍

File: CrossCutting/Authorization/JwtTokenGenerator.cs [hunk 2]
```diff
@@ -24,11 +25,16 @@ public class JwtTokenGenerator : IJwtTokenGenerator
...
+            {
+                { "isSU", true }
+            };
 
             var jwtToken = _tokenHandler.CreateToken(new SecurityTokenDescriptor
             {
                 Expires = now.AddDays(_jwtOptions.TokenLifeTime),
-                SigningCredentials = _signingCredentials
+                SigningCredentials = _signingCredentials,
+                Claims = payload
```
Comment (nreal) [inline CrossCutting/Authorization/JwtTokenGenerator.cs:37]: It is the same that` jwtToken.Payload.Add("isSU", true); `?

Comment (andresmoschini) [reply CrossCutting/Authorization/JwtTokenGenerator.cs:37]: I do not know, @msimionati could you share two deserialized tokens generated in these ways?

File: DopplerJobTest/TokenGeneratorTests.cs [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+        [Fact]
+        public async Task JwtTokenGenerator_ShouldBeOk_WhenIsSuTagIsPresentAndIsTrue()
+        {
+            // Arrange
+            var options = Options.Create(new JwtOptions()
+            {
+                TokenLifeTime = 1
+            });
+
+            var publicAndPrivate = new RSACryptoServiceProvider(2048);
```
Comment (nreal) [inline DopplerJobTest/TokenGeneratorTests.cs:24]: Great, maybe in the future, we can create a helper class for it

File: CrossCutting/Authorization/JwtTokenGenerator.cs (hunk 2 above)
Comment (msimionati) [reply CrossCutting/Authorization/JwtTokenGenerator.cs:37]: Apparently it's the same:

This is how it is generated as it is now.
HEADER:ALGORITHM & TOKEN TYPE
{
  "alg": "RS256",
  "typ": "JWT"
}
PAYLOAD:DATA
{
  "isSU": true,
  "nbf": 1605191023,
  "exp": 1605277420,
  "iat": 1605191023
}

This is how it is generated as a question @nreal (jwtToken.Payload.Add("isSU", true);)
HEADER:ALGORITHM & TOKEN TYPE
{
  "alg": "RS256",
  "typ": "JWT"
}
PAYLOAD:DATA
{
  "nbf": 1605190803,
  "exp": 1605277202,
  "iat": 1605190803,
  "isSU": true
}

Comment (nreal) [reply CrossCutting/Authorization/JwtTokenGenerator.cs:37]: Good

**Pull request merged**